pkg/roachprod/install/cockroach.go

```diff
@@ -744,6 +755,38 @@ func (c *SyncedCluster) shouldAdvertisePublicIP() bool {
 	return false
 }
 
+// createBackupSchedule creates a cluster backup schedule which runs an incremental every 15
+// minutes and a full every hour
+func (c *SyncedCluster) createBackupSchedule(
```

Comment:
Maybe rename to `createFixedBackupSchedule`? Let's also mention the hardcoded bucket in the comment to be clear that everything (i.e., schedule and bucket) is fixed for this iteration.